Lib/test/test_sys.py

```diff
@@ -1340,7 +1340,7 @@ def bar(cls):
             check(bar, size('PP'))
         # generator
         def get_gen(): yield 1
-        check(get_gen(), size('P2PPP4P4c8P2iciP'))
+        check(get_gen(), size('P2P4P4c8P2iciP'))
```

Comment:
Just a quibble, and not your fault, but this doesn't look right.
The total is presumably correct, but the order seems off. Shouldn't the stack use be at the end?

Reply:
Yeah, I don't know what this is.

Reply:
OK, let's not worry about for now.